components/DataTable/samples/DataTableVirtualizationSample.xaml

```diff
@@ -13,13 +13,12 @@
               VerticalAlignment="Top"
               ItemsSource="{x:Bind InventoryItems}">
         <ListView.Header>
-            <controls:DataTable Margin="12,0,0,0"
-                                ColumnSpacing="16">
+            <controls:DataTable ColumnSpacing="16">
```

Comment:
Maybe it's the missing margin? I lot of this is a bit fickle to line things up atm... 😟 Still not sure of a better approach between the different hosts yet. At least if we can have them setup in the samples to be decent it's a starting point.

Reply:
But then it's not the same for WASDK... 😭